cmd/releases.go

```diff
@@ -122,13 +122,14 @@ func runReleaseCreate(ctx *cli.Context) error {
 		if file, err = os.Open(asset); err != nil {
 			log.Fatal(err)
 		}
-		defer file.Close()
 
 		filePath := filepath.Base(asset)
 
 		if _, err = login.Client().CreateReleaseAttachment(owner, repo, release.ID, file, filePath); err != nil {
 			log.Fatal(err)
```

Comment:
file should be closed also closed before calling `log.Fatal(err)`

Reply:
Hm, if a file could not be opened, closing it also results in an error (`invalid argument`), I just tested that

Reply:
File will be opened in already other if. Here you will get this error if gitea server returns error for example. And in this case you need to close file also here before log.Fatal. Not in the if above where you check for error from `os.Open`

Reply:
```suggestion
			file.Close()
			log.Fatal(err)
```

Reply:
oh right I missed that. thanks